packages/material-ui/src/CssBaseline/CssBaseline.js

```diff
@@ -28,6 +28,9 @@ const useStyles = makeStyles(
           // Save printer ink.
           backgroundColor: theme.palette.common.white,
         },
+        '&::backdrop': {
```

Comment:
Ok, maybe we should go wit option n°3. What about adding a comment for document.body.requestFullScreen() ?

Reply:
Well, that wouldn't be aligned with the intention of setting the background color of the body thou.

Depending on how you see this, is a bug since Mui is taking responsibility to set up the background color.

I think that PR is just fine as it is right now.

But, up to you, I don't mind either way.

Reply:
Yes, I think it's an acceptable tradeoff (n°3). What about the comment that explain the focus on `document.body.requestFullScreen()` (it might not work with the other elements).

Reply:
```suggestion
         // Add support for document.body.requestFullScreen(), other elements, if transparent are not supported. 
        '&::backdrop': {
```